rules/falco-deprecated_rules.yaml

```diff
@@ -110,68 +134,93 @@
   items: [google.com, www.yahoo.com]
 
 - rule: Unexpected outbound connection destination
-  desc: > 
-    Detect any outbound connection to a destination outside of an allowed set of ips, networks, or domain names. 
-    This rule absolutely requires profiling your environment beforehand. Network-based rules are extremely crucial 
-    in any security program, as they can often provide the only definitive evidence. However, effectively operationalizing 
-    them can be challenging due to the potential for noise.
+  desc: >
+    Detect any outbound connection to a destination outside of an allowed
+    set of ips, networks, or domain names. This rule absolutely requires
+    profiling your environment beforehand. Network-based rules are extremely
+    crucial in any security program, as they can often provide the only
+    definitive evidence. However, effectively operationalizing them can be
+    challenging due to the potential for noise.
   condition: >
-    outbound 
+    outbound
     and not ((fd.sip in (allowed_outbound_destination_ipaddrs)) or
-             (fd.snet in (allowed_outbound_destination_networks)) or
-             (fd.sip.name in (allowed_outbound_destination_domains)))
+            (fd.snet in (allowed_outbound_destination_networks)) or
+            (fd.sip.name in (allowed_outbound_destination_domains)))
   enabled: false
-  output: Disallowed outbound connection destination (connection=%fd.name lport=%fd.lport rport=%fd.rport fd_type=%fd.type fd_proto=fd.l4proto evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty %container.info)
+  output: >
```

Comment:
@leogr you specifically didn't or don't like the `>` in general. What is is going to be going forward?

Reply:
@incertum @h4l0gen 

I prefer _plain flow strings_ with no style modifiers  (i.e., ie no `>` or `|`) 

YAML block scalars delimited with style modifiers are intended to preserve (|) or fold (>) newlines. We don't need that. On the other hand, with plain strings, trailing `\n`s are automatically trimmed (that's the preferable, IMO). Do you agree?

You can find here some example: https://yaml-multiline.info/

So I definitively prefer:
```yaml
output:
```
(without `>`)

Reply:
Yes, I am removing `>` modifiers from all tags. I am going to push changes today with line-length <130, after that let see what other changes we feel.